benches/reif.pl

```diff
@@ -40,8 +40,8 @@
 % Copy of _if/3, but with a different name, so it won't be expanded
 fif_(If_1, Then_0, Else_0) :-
     call(If_1, T),
-    (  T == true  -> call(Then_0)
-    ;  T == false -> call(Else_0)
+    (  T == true  -> Then_0
```

Comment:
Why, these variables will be wrapped with `call/1` during term-to-body conversion anyway

Reply:
No clear reason, it just looked more clean.

Reply:
With `call/1` it's no longer a **copy** of `if_/3`, making the comment wrong:

http://www.complang.tuwien.ac.at/ulrich/Prolog-inedit/sicstus/reif.pl